### PR TITLE
updated python client for new pulsar protos

### PR DESCRIPTION
**Pull request by rltvty**

### CODE REVIEW

File: clients/python/tests/test_formatters_shared.py [hunk 1]
```diff
@@ -80,7 +80,13 @@ def test_table_pulsar_config_html():
     namespace = "test-namespace"
     topic_name = "my-topic"
     pulsar = pulsar_pb.PulsarConfig(
-        **{"broker_service_url": broker_service_url, "auth_plugin": auth_plugin, "tenant": tenant, "namespace": namespace, "topic_name": topic_name }
+        **{
+            "broker_service_url": broker_service_url,
+            "auth_plugin": auth_plugin,
+            "tenant": tenant,
+            "namespace": namespace,
+            "topic_name": topic_name,
```
Comment (jordanrfrazier) [inline clients/python/tests/test_formatters_shared.py:88]: This omits the auth fields on purpose, correct?

File: clients/python/src/kaskada/materialization.py [hunk 2]
```diff
@@ -94,6 +98,10 @@ def __init__(
             namespace (str): pulsar namespace. defaults to "default".
             topic_name (str): final part of topic url. defaults to a randomly generated uuid
             broker_service_url (str): url to connect to pulsar broker. defaults to "pulsar://127.0.0.1:6650"
+            admin_service_url (str): the pulsar admin REST URL for the cluster. defaults to http://127.0.0.1:8080
+            auth_plugin (str): authentication plugin to use. e.g. "org.apache.pulsar.client.impl.auth.AuthenticationToken"
+            auth_params (str): authentication parameters. e.g. "token:xxx"
+            certificate_chain (str): a custom certificate chain to authenticate the server in TLS connections.
```
Comment (jordanrfrazier) [inline clients/python/src/kaskada/materialization.py:104]: Wren already maps these to sparrow correctly?

**Review by jordanrfrazier — APPROVED** [review]

File: clients/python/src/kaskada/materialization.py (hunk 2 above)
Comment (rltvty) [reply clients/python/src/kaskada/materialization.py:104]: wren passes destinations through to sparrow as-is.  doesn't do any mapping.

File: clients/python/tests/test_formatters_shared.py (hunk 1 above)
Comment (rltvty) [reply clients/python/tests/test_formatters_shared.py:88]: this is just a reformat of an existing test.  though I assume whomever created it wanted to omit auth fields on purpose.

**Pull request merged**